consoleApp/src/main/java/com/denis/consoleapp/service/PrintOrderCommand.java

```diff
@@ -0,0 +1,69 @@
+package com.denis.consoleapp.service;
+
+import com.denis.domain.Product;
+import com.denis.store.Store;
+
+import java.util.List;
+import java.util.Scanner;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+
+public class PrintOrderCommand implements Command {
+    Scanner scanner = new Scanner(System.in);
+    private Store store;
+    private int categoryNumber;
+    private int productNumber;
+
+    public PrintOrderCommand(Store store) {
+        this.store = store;
+    }
+
+    @Override
+    public void execute() {
```

Comment:
please think about refactoring of these method(class), cause now it looks like mess. some improvements: 1. move schedulers to separate method/class, 2. take a look at performance, cause you read data N times in cycles, you can optimize it, 3. printiing